js/form_editor_controller.js

```diff
@@ -1116,17 +1156,98 @@ class GlpiFormEditorController
         section.remove();
     }
 
+    /**
+     * Add a new comment block.
+     * @param {jQuery} target   Current position in the form
+     */
+    #addComment(target) {
+        let destination;
+        let action;
+
+        // Find the context using the target
+        if (
+            target.data('glpi-form-editor-question') !== undefined
+            || target.data('glpi-form-editor-comment') !== undefined
+        ) {
+            // Adding a new comment block after an existing question
+            destination = target;
+            action = "after";
+        } else if (target.data('glpi-form-editor-section') !== undefined) {
+            // Adding a comment block at the start of a section
+            destination = target
+                .closest("[data-glpi-form-editor-section]")
+                .find("[data-glpi-form-editor-section-blocks]");
+            action = "prepend";
+        } else if (target.data('glpi-form-editor-form') !== undefined) {
+            // Add a comment block at the end of the form
+            destination = $(this.#target)
+                .find("[data-glpi-form-editor-section]:last-child")
+                .find("[data-glpi-form-editor-section-blocks]:last-child");
+            action = "append";
+        } else {
+            throw new Error('Unexpected target');
+        }
```

Comment:
If this is the exact same logic than for questions, maybe we could move it to a common function.
This will help maintaining it in the future if we need to add some changes.